domainlab/utils/generate_benchmark_plots.py

```diff
@@ -143,28 +167,31 @@ def scatterplot_matrix(dataframe_in, file=None, reg=True, distinguish_param_setu
     distinguish_param_setups: if True the plot will not only distinguish between models,
         but also between the parameter setups
     '''
-    dataframe = dataframe_in.copy()
+    dataframe = dataframe_in.iloc[:, G_DF_PLOT_COL_START:].copy()
     index = list(range(5, dataframe.shape[1]))
     if distinguish_param_setups:
         dataframe_ = dataframe.iloc[:, index]
         dataframe_.insert(0, 'label',
                           dataframe[COLNAME_ALGO].astype(str) + ', ' +
                           dataframe[COLNAME_PARAM].astype(str))
+
+        g_p = sns.pairplot(data=dataframe_, hue='label', corner=True, kind=kind)
     else:
         index_ = list(range(5, dataframe.shape[1]))
         index_.insert(0, 0)
         dataframe_ = dataframe.iloc[:, index_]
 
-    if reg:
-        if not distinguish_param_setups:
-            g_p = sns.pairplot(data=dataframe_, hue=COLNAME_ALGO, corner=True, kind='reg')
-        else:
-            g_p = sns.pairplot(data=dataframe_, hue='label', corner=True, kind='reg')
-    else:
-        if not distinguish_param_setups:
-            g_p = sns.pairplot(data=dataframe_, hue=COLNAME_ALGO, corner=True)
-        else:
-            g_p = sns.pairplot(data=dataframe_, hue='label', corner=True)
+        g_p = sns.pairplot(data=dataframe_, hue=COLNAME_ALGO, corner=True, kind=kind)
+    # if reg:
```

Comment:
are we going to support this in the future? I am fine with first commenting it out if we would need this in the future version.

Reply:
I deleted the code. I think I do not need it anymore